### PR TITLE
RIP babel

### PR DESCRIPTION
**Pull request by MiniGod**
So, I decided to rip out babel. Node has come a long way and running babel server-side is kind of weird in my opinion as there is no build step. This should improve the speed of test and startup time.

Most of the lines changed are conversions from `import` to `require`, and `export default` to `module.exports`.

You guys probably do much more front-end stuff than me and are probably more used to using `import` and babel on a daily basis, so please let me know if you like this or not.

### CODE REVIEW

**Comment by koddsson** [comment]
I was thinking something really similar the other day! 💯

File: package.json [hunk 1]
```diff
@@ -46,9 +46,6 @@
   "dependencies": {
     "apis-helpers": "0.0.1",
     "axios": "^0.9.1",
-    "babel": "^6.5.2",
-    "babel-core": "^6.7.4",
-    "babel-preset-es2015": "^6.5.0",
```
Comment (MiniGod) [inline package.json]: Whatever I did, I couldn't get yarn to do what I wanted it to do, so I didn't commit the lock file. Can somebody else fix that for me?
(or ditch `yarn` completely and go with `npm@5` and `package-lock.json` files)

Comment (koddsson) [reply package.json]: I think `package-lock.json` files are the way to go but maybe in the next PR. Let me see if I can generate it.

Comment (koddsson) [reply package.json]: @MiniGod https://github.com/MiniGod/apis/pull/1

Comment (MiniGod) [reply package.json]: You should be able to push to my fork, but thanks a lot for the PR. I've merged. :)

**Pull request merged**